<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Live_window</class>
 <widget class="QDialog" name="Live_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Live Window</string>
  </property>
  <widget class="QSplitter" name="main_vertical_splitter">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>8</y>
     <width>929</width>
     <height>757</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <layout class="QHBoxLayout" name="content_layout">
     <item>
      <layout class="QVBoxLayout" name="control_layout">
       <item>
        <widget class="QPushButton" name="set_output_dir_button">
         <property name="text">
          <string>Set Output Directory</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="button_layout">
         <item>
          <widget class="QPushButton" name="start_experiment_button">
           <property name="styleSheet">
            <string notr="true">font-weight: bold;
font-size: 24px;
border-radius: 25px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stop_experiment_button">
           <property name="styleSheet">
            <string notr="true">background: transparent;
border-color: transparent;
border-radius: 50%;
font-weight: bold;
font-size: 24px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="vspacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QGridLayout" name="connection_settings_layout">
         <item row="0" column="0">
          <widget class="QLineEdit" name="vrpn_host_edit">
           <property name="text">
            <string>VRPN Host</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="vrpn_port_edit">
           <property name="text">
            <string>VRPN Port</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="sage_host_edit">
           <property name="text">
            <string>SAGE2 Host</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="sage_port_edit">
           <property name="text">
            <string>SAGE2 Port</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="vspacer_1">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QGridLayout" name="camera_settings_grid">
         <item row="0" column="0">
          <widget class="QLabel" name="video_device_index_label">
           <property name="text">
            <string>Video Device Index</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="video_device_index_edit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="frame_rate_label">
           <property name="text">
            <string>Frame Rate:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="frame_rate_value_label">
           <property name="styleSheet">
            <string notr="true">font-weight: bold;</string>
           </property>
           <property name="text">
            <string> Unknown</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="frame_num_label">
           <property name="text">
            <string>Frame Number:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="frame_num_value_label">
           <property name="styleSheet">
            <string notr="true">text-align: right;</string>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="hspacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="video_feed_label">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: transparent;</string>
       </property>
       <property name="text">
        <string>Video Stream</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hspacer_1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <layout class="QVBoxLayout" name="bottom_layout" stretch="0">
     <item>
      <widget class="QTextBrowser" name="log_window">
       <property name="styleSheet">
        <string notr="true">background-color: #e6e8ed;
font-weight: bold;
border-radius: 5px;
border-width: 2px;
border-style: inset;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
